README.md

```diff
@@ -34,10 +34,15 @@ own and can obtain certificates for.
   1. Install Go version 1.10 or higher. Optionally, set
      [$GOPATH](https://github.com/golang/go/wiki/GOPATH) to something (default
      is `~/go`) and/or add `$GOPATH/bin` to `$PATH`.
-  2. `go get -u -mod=vendor github.com/ampproject/amppackager/cmd/amppkg`
+  1. `go get -u -mod=vendor github.com/ampproject/amppackager/cmd/amppkg`
```

Comment:
(optional) For now, I'd be OK with putting an if statement here based on `go version`, until we work out the details.

Reply:
Done.